frontend/src/components/Footer.jsx

```diff
@@ -7,18 +7,22 @@ const Footer = () => {
     const [t] = useTranslation(['mainnav']);
 
     return (
-        <FooterContainer>
-            <EpiIcon>{t('footer.E')}</EpiIcon>
-            <TextStyles>
-                {t('footer.epimetheus')} {t('footer.version')}{' '}
-                <span className="underline">{packageJson.version}</span>{' '}
-                {t('footer.powered')}{' '}
-                <span className="underline">
-                    <a href="https://www.siili.com">{t('footer.siili')}</a>
-                </span>
-                .
-            </TextStyles>
-        </FooterContainer>
+        <footer>
+            <FooterContainer>
+                <EpiIcon role="img" aria-label="Small Epimetheus logo">
+                    {t('footer.E')}
+                </EpiIcon>
```

Comment:
Is this really something that is needed to be seen/read by screen readers? In my opinion it's purely decorative.

Reply:
I'll change this to aria-hidden